lametro/events.py

```diff
@@ -88,6 +109,8 @@ def api_events(self, *args, **kwargs):
                 partner_event = self._find_partner(unpaired_event)
                 if partner_event is not None:
                     yield partner_event
+                else:
```

Comment:
historic events don't have spanish partners, so i suspect this will break for full event scrapes. what do you think about checking the date on the event to confirm we should see a spanish event before raising an exception? (check in legistar to check the date when spanish events started to appear. i think it was last summer...)

Reply:
i added in a check!